review/fileToReview.js

```diff
@@ -12,62 +11,62 @@ var domain = "bank.local.fr"
+            }
+        } else {
+            headers = {
+                "Authorisation": authorization,
+                "Content-type": "application/json",
+                "Accept": "application/json",
+            }
+        }
 
-	  var {code, response } = await doRequest('GET',
-      domain + '/accounts/'+ id + '/transactions?' + `page=${page}`,
-      headers);
+        var {code, response} = await doRequest('GET',
+            domain + '/accounts/' + id + '/transactions?' + `page=${page}`,
+            headers);
 
 
-		if (response && code == 200 && response.data) {
-      if (response.data.meta) {
-        if (response.data.meta.hasPageSuivante) {
-          let mouvements = response.data.Mouvements;
-          var date = mouvements[mouvements.length -1].dateValeur;
-          if (date <= fromDate) {
-            console.log("FromDate is Reached - we don't need more transaction");
-          } else {
-            // if we have mouvements
-            if (mouvements) {
-              if (assertTransactions(mouvements)) {
-                return [];
-              } else {
-                console.log(`Push transactions from page ${page}`);
-              }
-            } else {
-              throw new Error("Empty list of transactions ! " + JSON.stringify(previousTransactions));
+        if (response && code == 200 && response.data) {
+            if (response.data.meta) {
+                if (response.data.meta.hasPageSuivante) {
+                    let mouvements = response.data.Mouvements;
+                    var date = mouvements[mouvements.length - 1].dateValeur;
+                    if (date <= fromDate) {
+                        console.log("FromDate is Reached - we don't need more transaction");
+                    } else {
+                        // if we have mouvements
+                        if (mouvements) {
+                            if (assertTransactions(mouvements)) {
+                                return [];
+                            } else {
+                                console.log(`Push transactions from page ${page}`);
+                            }
+                        } else {
+                            throw new Error("Empty list of transactions ! " + JSON.stringify(previousTransactions));
+                        }
+                        let nextPagesTransactions = fetchTransactions(fromDate, authorization, (jws || null), id, page + 1, mouvements);
+                        response.data.Mouvements = mouvements.concat(nextPagesTransactions);
+                    }
+                }
             }
-            let nextPagesTransactions = fetchTransactions(fromDate, authorization, (jws || null), id, page + 1, mouvements);
-            response.data.Mouvements = mouvements.concat(nextPagesTransactions);
-          }
-        }
-      }
-      return response.data.Mouvements;
-    } else throw new Error();
+            return response.data.Mouvements;
+        } else throw new Error();
```

Comment:
try to think by the negation to avoid the use of else
it will help improve the code readability